teps/0001-tekton-enhancement-proposal-process.md

```diff
@@ -308,6 +309,12 @@ for withdrawal.
 A TEP can be moved to the `implementable` state if it doesn't need
 any more discussion and is approved as is.
 
+A TEP can be marked `deferred` if it is not actively being worked on and
+there are open questions. A `deferred` TEP can be moved to another state
+with an explanation. If we answer the open questions and pick up the TEP,
```

Comment:
> A `deferred` TEP can be moved to another state with an explanation.

`proposed` -> `deferred` cannot be assigned `implementable`.

Reply:
it seems this is aligned with my comment below that a `deferred` TEP cannot be marked `implementable` without updates to it